uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_absence-tab-page.scss

```diff
@@ -7,4 +7,8 @@
   .tab-pane {
     padding: 0 !important;
```

Comment:
Is "!important" necessary? Does that mean that the bootstrap theme is applied afterwards and without important it will not work? Is there a way to avoid "!important" in this case?